Xamarin.Forms.ControlGallery.Android/ChangeDensityButtonRenderer.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Util;
+using Android.Views;
+using Android.Widget;
+using Xamarin.Forms;
+using Xamarin.Forms.Controls;
+using Xamarin.Forms.ControlGallery.Android;
+using Xamarin.Forms.Platform.Android;
+
+[assembly: ExportRenderer(typeof(ChangeDensityButton), typeof(ChangeDensityButtonRenderer))]
+namespace Xamarin.Forms.ControlGallery.Android
+{
+	public class ChangeDensityButtonRenderer : Xamarin.Forms.Platform.Android.FastRenderers.ButtonRenderer
+	{
+		public ChangeDensityButtonRenderer(Context context) : base(context)
+		{
+		}
+
+		protected override void OnElementChanged(ElementChangedEventArgs<Button> e)
+		{
+			Element.Clicked -= ChangeDensityClicked;
+			Element.Clicked += ChangeDensityClicked;
```

Comment:
What purpose does this serve exactly?

Reply:
Here i create this click event on android plataform to change context density at runtime, simulating device density change...
Maybe i can create some better like a command property that is setted at android plataform...


I'm open to suggestions 😃